show/main.py

```diff
@@ -1383,8 +1388,20 @@ def runningconfiguration():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def all(verbose):
     """Show full running configuration"""
-    cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    cmd = ['sonic-cfggen', '-d', '--print-data']
+    stdout = get_cmd_output(cmd)
```

Comment:
>stdout [](http://example.com/codeflow?start=4&length=6)

suggest function return `(stdout, rc)`. After calling this function, check `rc` first.

Reply:
@qiluo-msft  If we follow the original behavior, we don't need to check the rc. As it will be empty string by default if rc is non-zero.
```
def run_command(command, display_cmd=False, return_cmd=False):
    if display_cmd:
        click.echo(click.style("Command: ", fg='cyan') + click.style(command, fg='green'))

    # No conversion needed for intfutil commands as it already displays
    # both SONiC interface name and alias name for all interfaces.
    if clicommon.get_interface_naming_mode() == "alias" and not command.startswith("intfutil"):
        clicommon.run_command_in_alias_mode(command)
        raise sys.exit(0)

    proc = subprocess.Popen(command, shell=True, text=True, stdout=subprocess.PIPE)

    while True:
        if return_cmd:
            output = proc.communicate()[0]                   <===============
            return output
```